bears/php/PHPStanBear.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='phpstan',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)  (?P<message>.*)')
+class PHPStanBear:
+    """
+    Checks the code with ``phpstan analyze``.
+    This can run it on multiple files and folders.
+    See <https://github.com/phpstan/phpstan> for more information.
+    """
+    LANGUAGES = {'PHP'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('analyse', '--no-ansi', filename)
```

Comment:
@damngamerz You must be having some bugs in your code that are causing this. Please read our Developer's guide thoroughly as most of the things we generally need are mentioned over there, read the documentation of PHPStan to know how exactly their config file works and lastly, in case of some error feel free to discuss on the Gitter channel (as I might miss your notification others will still be able to help you out)